packml_msgs/msg/ItemizedStats.msg

```diff
@@ -0,0 +1,6 @@
+# This message contains itemized statics for PackML statistics reporting
+# Each item containds a unique
+
+int16 id                          # unique identifier (user defined)
+int32 count                       # number of occurances
```

Comment:
`PackML_Tag_Naming_Guidelines_V2.0.pdf` seems to suggest that PackML uses 32bit unsigned integers whenever `Integer` is given as the tag type (section 3.5). Is the use of `int32` here on purpose?